workflows/long-read-mngs/run.wdl

```diff
@@ -716,11 +872,47 @@ workflow czid_long_read_mngs {
         reads_fastq = RunSubsampling.subsampled_fastq,
         m8 = RunCallHitsNR.deduped_out_m8,
         hitsummary = RunCallHitsNR.hitsummary,
-        reads_to_contigs_sam = RunReadsToContigs.reads_to_contigs_sam,
-        db_type = "nr",
+        reads_to_contigs_tsv = RunReadsToContigs.reads_to_contigs_tsv,
+        docker_image_id = docker_image_id,
+    }
+
+    call ReassignM8NT {
+      input:
+        m8 = RunCallHitsNT.deduped_out_m8,
+        reads_to_contigs_tsv = RunReadsToContigs.reads_to_contigs_tsv,
         docker_image_id = docker_image_id,
     }
 
+    call ReassignM8NR {
+      input:
+        m8 = RunCallHitsNR.deduped_out_m8,
+        reads_to_contigs_tsv = RunReadsToContigs.reads_to_contigs_tsv,
+        docker_image_id = docker_image_id,
+    }
+
+    call GenerateAnnotatedFasta {
+      input:
+        pre_alignment_fasta = RunSubsampling.subsampled_fastq,
```

Comment:
The function is supposed to take in a fasta but the output of subsampling is a fastq. we could use `https://bioinf.shenwei.me/seqkit/usage/#fq2fa` to convert

Reply:
Thanks!